### PR TITLE
Add updated API doc and basic timeseries commands

### PR DESCRIPTION
**Pull request by bnaecker**
- Implement a few bits for timeseries commands under the `experimental` heading
- Add a basic dashboard that displays scalar timeseries based on a query

### CODE REVIEW

File: cli/src/cli_builder.rs [hunk 1]
```diff
@@ -500,7 +504,8 @@ fn xxx<'a>(command: CliCommand) -> Option<&'a str> {
         | CliCommand::SiloMetric
         | CliCommand::SystemMetric
         | CliCommand::UserBuiltinList
-        | CliCommand::UserBuiltinView => None,
+        | CliCommand::UserBuiltinView
+        | CliCommand::PhysicalDiskView => None,
```
Comment (ahl) [inline cli/src/cli_builder.rs]: can we add this? If we don't I'll never find it again. "system hardware disk view" and put it with its friends?

**Review by ahl — APPROVED** [review]
one change; then good to go

File: cli/src/cli_builder.rs (hunk 1 above)
Comment (bnaecker) [reply cli/src/cli_builder.rs]: Done in 22645eb:

```
bnaecker@shale : ~/oxide.rs $ ./target/release/oxide system hardware disk view
error: the following required arguments were not provided:
  --disk-id <disk-id>

Usage: oxide system hardware disk view --disk-id <disk-id>

For more information, try '--help'.
```

Comment (bnaecker) [reply cli/src/cli_builder.rs]: Tests fixed in 671c3e1

**Pull request merged**